etc/security/mh_default_org.xml

```diff
@@ -261,6 +261,9 @@
     <sec:intercept-url pattern="/static/**" method="GET" access="ROLE_ANONYMOUS" />
     <sec:intercept-url pattern="/export/**" method="GET" access="ROLE_ANONYMOUS" />
 
+    <!-- Enable admin access to Opencast Studio -->
+    <sec:intercept-url pattern="/studio/**" access="ROLE_STUDIO" />
```

Comment:
No one, even the global admin, has this permission by default.

Reply:
The global admin can always access this. Did you get refusal as admin?

Reply:
Logged in as admin:opencast and got a 403.

Reply:
Interesting. Anyway, added `ROLE_ADMIN`.